src/api/v2/models/job.ts

```diff
@@ -53,6 +53,10 @@ export interface IJobListQeury {
     withTotalCount?: boolean;
 }
 
+export interface IEventListQuery {
+    type?: string; // "Warning" or "Normal"
```

Comment:
If you are sure the type will be `"Warning", "Normal" or undefined`,
```suggestion
    type?: 'Warning' | 'Normal';
```

Reply:
ok. thanks for the correction.